grafana-formula/grafana-formula.spec

```diff
@@ -44,7 +43,6 @@ cp -R grafana/* %{buildroot}%{fdir}/states/%{fname}
 cp -R metadata/* %{buildroot}%{fdir}/metadata/%{fname}
 
 %files
-%defattr(-,root,root)
```

Comment:
same here. spec-cleaner is sometimes removing too much. not sure  if this is wanted or  no

Reply:
Indeed, I think it's safer to keep this one. I reverted the change.